content/en/blog/_posts/2020-12-10-Granular-Control-of-Volume-Permission-Changes.md

```diff
@@ -0,0 +1,56 @@
+# Granular Control of Volume Permission Changes
+
```

Comment:
These lines need to be removed. 
A blog post has to start with the metadata as noted [here](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#technical-considerations-for-submitting-a-blog-post)

Reply:
Thanks!  I've fixed that, and am fixing the headers now so that they don't match the Title.